### DISCUSSION TITLE
apk upgrade --no-cache -a --prune

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#4491] [Ideas] [answered]
Since alpine sometimes updates apk packages included by default in the alpine docker image (because of CVEs or other reasons), without updating the docker image, we should run `apk upgrade --no-cache -a --prune` as the first line each Dockerfile, for the debian based images I think we should do the same (bit with apt)

**Comment by szaimen** [comment]
Hm... honestly I am not completely sure about this as dependencies are currently not pinned already and in that case it gets even worse...

**Reply by szaimen** [reply]
Made a typo and corrected to *pinned instead of fixed

**Reply by Zoey2936** [reply]
but this line would only affect the preinstalled apks? apks we installed are always the newest version, even without this line

**Reply by szaimen** [reply]
Right. Would you mind opening a PR that implements this?

**Comment by szaimen** [comment, marked as answer]
Will be done in https://github.com/nextcloud/all-in-one/pull/4559
